packages/calcite-components/src/components/sort-handle/sort-handle.e2e.ts

```diff
@@ -0,0 +1,160 @@
+  });
+
+  describe("accessible", () => {
+    accessible(`<calcite-sort-handle label="test" set-position="4" set-size="10"></calcite-sort-handle>`);
+  });
+
+  it("sets handle tooltip", async () => {
+    const page = await newE2EPage();
+    const label = "Hello World";
+    await page.setContent(
+      `<calcite-sort-handle lang="en" label="${label}" set-position="4" set-size="10"></calcite-sort-handle>`,
+    );
+    await page.waitForChanges();
+
+    const handle = await page.find("calcite-sort-handle");
+    await handle.callMethod("setFocus");
+    const button = await page.find(`calcite-sort-handle >>> .${CSS.handle}`);
+    const messages: SortHandleMessages = await handle.getProperty("messages");
+
+    expect(await button.getProperty("title")).toBe(
+      messages.repositionLabel
+        .replace(SUBSTITUTIONS.label, label)
+        .replace(SUBSTITUTIONS.position, "4")
+        .replace(SUBSTITUTIONS.total, "10"),
+    );
+  });
+
+  it("fires calciteSortHandleReorder event", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-sort-handle label="test" set-position="4" set-size="10"></calcite-sort-handle>`);
+    await skipAnimations(page);
+
+    const sortHandle = await page.find("calcite-sort-handle");
+
+    const calciteSortHandleReorderSpy = await page.spyOnEvent("calciteSortHandleReorder");
+
+    const action = await page.find(`calcite-sort-handle >>> .${CSS.handle}`);
+    await action.callMethod("setFocus");
+
+    await page.keyboard.press("ArrowDown");
+    await page.waitForChanges();
+    expect(await sortHandle.getProperty("open")).toBe(true);
+
+    await page.keyboard.press("Enter");
+    await page.waitForChanges();
+    await page.waitForChanges();
+    expect(await calciteSortHandleReorderSpy.lastEvent.detail.reorder).toBe(REORDER_VALUES[0]);
+    expect(calciteSortHandleReorderSpy).toHaveReceivedEventTimes(1);
+  });
+
+  it("fires calciteSortHandleMove event", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-sort-handle label="test" set-position="4" set-size="10"></calcite-sort-handle>`);
+    await skipAnimations(page);
+
+    const moveToItems = [
+      { label: "List 2", id: "list2" },
+      { label: "List 3", id: "list3" },
+    ];
+
+    const sortHandle = await page.find("calcite-sort-handle");
+    sortHandle.setProperty("moveToItems", moveToItems);
+    await page.waitForChanges();
+
+    const calciteSortHandleMoveSpy = await page.spyOnEvent("calciteSortHandleMove");
+
+    const action = await page.find(`calcite-sort-handle >>> .${CSS.handle}`);
+    await action.callMethod("setFocus");
+
+    await page.keyboard.press("ArrowUp");
+    await page.waitForChanges();
+    expect(await sortHandle.getProperty("open")).toBe(true);
+
+    await page.keyboard.press(" ");
+    await page.waitForChanges();
+    await page.waitForChanges();
```

Comment:
Is this extra `page.waitForChanges` intentional?

Reply:
Will remove